docs/installing-packages.md

```diff
@@ -0,0 +1,98 @@
+# Installing packages
+This documentation dives deeper into how a user can configure packages for installation inside the EIB image. Furthermore, it explains the **RPM resolution** process that EIB goes through so it can ensure that configured packages will be successfully installed even in an **air-gapped** environment.
+
+## Supported systems
+EIB's **RPM resolution** process and package installation has been tested on the following `x86_64` systems: 
+1. [SLES 15-SP5](https://www.suse.com/download/sles/)
+1. [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/)
+1. [Ubuntu 22.04](https://releases.ubuntu.com/jammy/)
+1. [Fedora Linux](https://fedoraproject.org/server/download)
+
+## Specify packages for installation
+You can configure packages for installation in the following ways:
+1. provide a `packageList` configuration under `operatingSystem.packages` in the EIB image configuration file
+1. create a `rpms` directory under EIB's configuration directory and provide local RPM files that you want to be installed on the image
+
+### Install packages through 'packageList'
+To install a package using the `packageList` configuration, at a minimum you must configure the following under `operatingSystem.packages`:
+1. valid package names under `packageList`
+1. `additionalRepos` or `sccRegistrationCode`
```

Comment:
Nit: This is user facing documentation mostly so more details can be helpful. Ignore if you prefer it as it currently is.
```suggestion
1. Valid package names under `packageList`
1. Valid RPM repositories under `additionalRepos` or an SCC registration code under `sccRegistrationCode`
```